### PR TITLE
@ungap/structured-clone: add `json` option; add `json` submodule

### PR DESCRIPTION
**Pull request by benine203**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Package source code](https://github.com/ungap/structured-clone/blob/main/esm/json.js)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/ungap__structured-clone/json.d.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+     */
+    function stringify(any: any): string;
+
+    /**
+     * Revive a previously stringified structured clone.
+     * @param {string} str previously stringified data as string.
+     * @returns {any} whatever was previously stringified as clone.
+     */
+    function parse(str: string): any;
+}
```
Comment (peterblazejewicz) [inline types/ungap__structured-clone/json.d.ts:16]:
```suggestion
/**
 * Represent a structured clone value as string.
 * @param {any} any some clone-able value to stringify.
 * @returns {string} the value stringified.
 */
export function stringify(any: any): string;

/**
 * Revive a previously stringified structured clone.
 * @param {string} str previously stringified data as string.
 * @returns {any} whatever was previously stringified as clone.
 */
export function parse(str: string): any;
```
(outcome is same IMO, there is no other types exported requiring namespace in module)

Comment (benine203) [reply types/ungap__structured-clone/json.d.ts:16]: I will apply this when I am on kbd next. Thank you so very much.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@benine203 thanks!

**Comment by ebenali** [comment]
Ready to merge

**Comment by benine203** [comment]
Ready to merge

**Pull request merged**